scripts/benchmarking/msdnet_benchmarks.py

```diff
@@ -0,0 +1,144 @@
+#%% This example shows how to train MSDNet for full angle, noisy measurements.
```

Comment:
This folder is for a paper we did on bechmarking. Bad name, but I haven't gotten around to fix it yet. This should be in a different folder, but not sure where yet.

Reply:
added some extra folders, probably worth having a clean through example_scripts at some point. It's become just a bit of a dump of whatever doesn't fit anywhere else (atleast that's how I've been using it, oops). Suppose that's for later down the line when proper demos get made though.